rsconnect/main.py

```diff
@@ -1072,7 +1072,9 @@ def deploy_html(
     api_key: str = None,
     insecure: bool = False,
     cacert: typing.IO = None,
+    exclude=None,
```

Comment:
It doesn't seem great to have both `exclude` and `excludes` here. Can we standardize on `exclude` to match all of the other deployment commands?